tests/MongoDB.Driver.Tests/MongoClientSettingsTests.cs

```diff
@@ -493,7 +493,6 @@ public void TestFromUrlWithMongoDBX509_without_username()
         }
 
         [Theory]
-        [InlineData("mongodb+srv://username:password@test5.test.build.10gen.cc/?connect=direct", ConnectionStringScheme.MongoDB, "localhost.test.build.10gen.cc:27017")]
```

Comment:
This test fails with the new requirement regarding directConnection. They're described here:https://github.com/mongodb/specifications/commit/e56f5eceed7729f8b9b43a4a1f76c7e5840db49f#diff-5e688c46cf9e7c28e7b4b821cf4cd145R66